llama_index/chat_engine/simple.py

```diff
@@ -0,0 +1,77 @@
+from typing import Any, Optional
+
+from llama_index.chat_engine.types import BaseChatEngine, ChatHistoryType
+from llama_index.chat_engine.utils import to_chat_buffer
+from llama_index.indices.service_context import ServiceContext
+from llama_index.prompts.base import Prompt
+from llama_index.response.schema import RESPONSE_TYPE, Response
+
+DEFAULT_TMPL = """\
```

Comment:
out of curiosity did you come up with this prompt

Reply:
It's a mash up of prompts found online, made concise by ChatGPT LOL